src/app/core/interfaces/basic-rundown-service.ts

```diff
@@ -0,0 +1,6 @@
+import {Observable} from 'rxjs';
+import {BasicRundown} from "../models/basic-rundown";
+
+export abstract class BasicRundownService {
```

Comment:
It should. They are not interfaces, so it doesn't make sense that they are in a folder called "interfaces".

> Because that is needed for us to map a "interface" to an implementation when doing dependency injection.

And that doesn't work with regular interfaces?

Reply:
No. Dependency injection is a runtime concept. When we compile the code from Typescript to Javascript, most type information, including the entire concept of interfaces, is stripped away.